underdocs-renderer/src/main/kotlin/underdocs/renderer/output/html/render/page/PageRenderer.kt

```diff
@@ -57,8 +60,26 @@ class PageRenderer(private val linker: Linker,
                             .withHref(linker.siteLinkBetween(visitable, "_static/code.css"))
             ),
             body(
+                    script()
+                            .with(UnescapedText("""
+                                (function () {
+                                  if (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) {
+                                    document.body.classList.add('night');
+                                  }
+                                })();
```

Comment:
Although it is a common practice to leave IIFE-s anonymous, I'd advise against doing so. If an error occurs inside an anonymous IIFE, then it is much harder to understand the stack trace, because the function will appear as "anonymous" or "f()". 
To ease the debugging experience, would you please consider naming the IIFE-s on this page?